src/Sentry/Span.cs

```diff
@@ -73,9 +76,11 @@ public class Span : ISpan, IJsonSerializable
         /// <summary>
         /// Initializes an instance of <see cref="Span"/>.
         /// </summary>
-        public Span(Transaction parentTransaction, SpanId? parentSpanId, string operation)
+        public Span(IHub hub, Transaction parentTransaction, SpanId? parentSpanId, string operation)
```

Comment:
Once we go with the SentryTracer approach taken by @brustolin we can drop this requirement of having IHub in Span